behave_django/management/commands/behave.py

```diff
@@ -21,45 +21,63 @@ def add_command_arguments(parser):
         help="Don't create a test database. USE AT YOUR OWN RISK!",
     )
     parser.add_argument(
-        '--keepdb',
+        '-k', '--keepdb',
         action='store_true',
         default=False,
         help="Preserves the test DB between runs.",
     )
 
 
-def add_behave_arguments(parser):
+def add_behave_arguments(parser):  # noqa
     """
     Additional command line arguments extracted directly from behave
     """
 
+    # Option strings that conflict with Django
     conflicts = [
         '--no-color',
-        '--version'
+        '--version',
+        '-c',
+        '-k',
+        '-v'
     ]
 
+    parser.add_argument(
+        'paths',
+        action='store',
+        nargs='*',
+        help="Feature directory, file or file location (FILE:LINE)."
+    )
+
     for fixed, keywords in behave_options:
-        # TODO: accept short options too
         keywords = keywords.copy()
-        long_option = None
+
+        # Configfile only entries are ignored
+        if not fixed:
+            continue
+
+        # Build option strings, not including conflicting option strings
+        option_strings = []
         for option in fixed:
-            if option.startswith("--"):
-                long_option = option
-                break
+            if option in conflicts:
+                continue
+
+            option_strings.append(option)
 
-        # Do not add conflicting options
-        if long_option in conflicts:
+        # No use in adding an option if it doesn't have an option string
+        if len(option_strings) == 0:
```

Comment:
How about  ... instead?
``` python
if not option_strings:
```
I have been told this is more pythonic.

Also, consider rewording the comment above to make it sound more positive. Or, even better, omit it if the code is self-explanatory.